doc/source/ray-contribute/development.rst

```diff
@@ -363,6 +363,8 @@ run the following (via ``-c`` ``fastbuild``/``dbg``/``opt``, respectively):
 
 .. code-block:: shell
 
+ # Ray is sensitive to the Bazel version. Please ensure that you use the same
+ # version of Bazel as the one installed by ci/env/install-bazel.sh.
```

Comment:
people should install and use `bazelisk` instead.

we should also change all our `bazel` usage in docs into `bazelisk`, and then we do not need this comment.